client/components/aboutUs/styles.js

```diff
@@ -0,0 +1,122 @@
+import styled, { css } from "styled-components";
+import { breakpoint } from "../../frontend-config";
+
+import { Heading2, TextBody } from "../shared/styles";
+
+export const Wrapper = styled.div`
+  background: ${props => props.theme.colors.light};
+  padding: 30px 30px 50px;
+  @media (min-width: ${breakpoint("lg")}) {
+    padding: 55px 30px;
+  }
+`;
+
+export const SubInfo = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  padding: 55px 20px;
+
+  a {
+    color: ${props => props.theme.colors.link};
+  }
+
+  svg {
+    margin-bottom: 45px;
+    width: 40%;
+  }
+
+  @media (min-width: ${breakpoint("xl")}) {
+    flex-direction: row;
+    gap: 60px;
+    svg {
+      margin-bottom: 0px;
+    }
+  }
+
+  ${props =>
+    props.grey &&
+    css`
+      background: ${props => props.theme.colors.grey};
```

Comment:
Since `props` has been brought in here at line 40, the value in line 43 can just be `${props.theme.colors.grey}`